### PR TITLE
Update Requirements article for Umbraco 11

### PR DESCRIPTION
**Pull request by sofietoft**
Updated
- .NET version to **7.0**
- Link to Microsoft documentation (line 56)

Are there any updated requirements in terms of local development or SQL?

### CODE REVIEW

File: Fundamentals/Setup/Requirements/index-v11.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+* Chrome (Latest)
+* Edge (Chromium)
+* Firefox (Latest)
+* Safari (Latest)
+
+## Local Development
+
+* Either OS:
+  * Microsoft Windows 7 SP1, 8.1, 10 and 11
```
Comment (Jeavon) [inline Fundamentals/Setup/Requirements/index-v11.md:19]: I don't believe Windows 7 or 8.1 are supported by .NET7 - https://github.com/dotnet/announcements/issues/226

File: Fundamentals/Setup/Requirements/index-v11.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  * MacOS High Sierra 10.13
+  * Linux (Ubuntu, Alpine, CentOS, Debian, Fedora, openSUSE and other major distributions)
+* One of the following .NET Tools or Editors:
+  * [Visual Studio Code](https://code.visualstudio.com/) with the [IISExpress extension](https://marketplace.visualstudio.com/items?itemName=warren-buckley.iis-express)
+  * [Microsoft Visual Studio](https://www.visualstudio.com/) 2022
```
Comment (Jeavon) [inline Fundamentals/Setup/Requirements/index-v11.md:24]: v17.4 minimum is required for .net7 RC2 - currently, this is available only on preview edition

File: Fundamentals/Setup/Requirements/index-v11.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  * [JetBrains Rider](https://www.jetbrains.com/rider) **version 2020.3 and higher**
```
Comment (Zeegaan) [inline Fundamentals/Setup/Requirements/index-v11.md:25]: This should be 2022.3 👍

File: Fundamentals/Setup/Requirements/index-v11.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+You can use [Umbraco Cloud](https://umbraco.com/products/umbraco-cloud/) to manage the hosting infrastructure. All Umbraco Cloud plans are hosted on Microsoft Azure, which gives your site a proven and solid foundation.
+:::
+
+### Miminium requirements to run Umbraco
+
+Umbraco can run on an environment with the minimum requirements stated below:
+
+* Windows Server 2012 R2 and higher
+* IIS 8.5 and higher
+* SQL Server 2012 and higher
```
Comment (Zeegaan) [inline Fundamentals/Setup/Requirements/index-v11.md:52]:
```suggestion
* SQL Server 2012 and higher / SQLite 3.38.0 or higher
```

**Review by Zeegaan — CHANGES_REQUESTED** [review]
Had a few things besides the two points @Jeavon brings up 👍

**Review by Zeegaan — APPROVED** [review]
After the changes it looks good to me 🎉

**Comment by sofietoft** [comment]
Thanks @Zeegaan and @Jeavon !

I'll merge this one in then 👏

**Comment by Jeavon** [comment]
All good!

**Pull request merged**